src/multiview_stitcher/registration.py

```diff
@@ -450,12 +450,35 @@ def get_node_params_from_reg_graph(g_reg):
 
 
 def register(
-    msims,
-    reg_channel_index=None,
-    transform_key=None,
-    new_transform_key=None,
-    registration_binning=None,
-):
+    msims: [multiscale_spatial_image],
+    reg_channel_index: int = None,
+    transform_key: unknown = None,
+    new_transform_key: unknown = None,
+    registration_binning: unknown = None,
+) -> [parametes]:
```

Comment:
Found a typo here :)

Reply:
I will fix the typo, but this was just for me to have a test case for the documentation. I think the return type should also be more explanatory i.e. _transformation matrix 4x4_ or whatever it is.

Reply:
Agreed.